build-system/tasks/release/index.js

```diff
@@ -0,0 +1,430 @@
+
+  const tarWritableStream = tar.extract({
+    cwd: ampSwTempDir,
+    filter: (path) => path.startsWith('package/dist'),
+    strip: 2, // to strip "package/dist/".
+  });
+  (await fetch(ampSwTarballUrl)).body.pipe(tarWritableStream);
+  await new Promise((resolve) => {
+    tarWritableStream.on('end', resolve);
+  });
+
+  await Promise.all(
+    distFlavorTypes.map((flavorType) =>
+      fs.copy(ampSwTempDir, path.join(tempDir, flavorType, 'dist/sw'))
+    )
+  );
+
+  logSeparator_();
+}
+
+/**
+ * Copies compiled build artifacts to RTV-based directories.
+ *
+ * Each flavor translates to one or more RTV numbers, for a detailed explanation
+ * see spec/amp-framework-hosting.md.
+ *
+ * @param {!Array<!Object>} distFlavors list of AMP flavors to build.
+ * @param {string} tempDir full directory path to temporary working directory.
+ * @param {string} outputDir full directory path to emplace artifacts in.
+ */
+async function populateOrgCdn_(distFlavors, tempDir, outputDir) {
+  for (const {flavorType, rtvPrefixes} of distFlavors) {
+    await Promise.all(
+      rtvPrefixes.map(async (rtvPrefix) => {
+        const rtvNumber = `${rtvPrefix}${VERSION}`;
+        const rtvPath = path.join(outputDir, 'org-cdn/rtv', rtvNumber);
+        await fs.ensureDir(rtvPath);
+        return fs.copy(path.join(tempDir, flavorType, 'dist'), rtvPath);
+      })
+    );
+  }
+
+  logSeparator_();
+}
+
+/**
+ * Prepends the AMP_CONFIG configuration object to all the entry files.
+ *
+ * Entry files are those that publishers would embed in their page, i.e.
+ * https://cdn.ampproject.org/v0.js, but could be others for different users,
+ * e.g., /amp4ads-v0.js for AMP ads.
+ *
+ * @param {string} outputDir full directory path to emplace artifacts in.
+ */
+async function prependConfig_(outputDir) {
+  const activeChannels = Object.entries(CHANNEL_CONFIGS).filter(
+    ([rtvPrefix]) => {
+      const rtvNumber = `${rtvPrefix}${VERSION}`;
+      const rtvPath = path.join(outputDir, 'org-cdn/rtv', rtvNumber);
+      return fs.pathExistsSync(path.join(rtvPath));
+    }
+  );
+
+  const allPrependPromises = [];
+  for (const [rtvPrefix, channelConfig] of activeChannels) {
+    const rtvNumber = `${rtvPrefix}${VERSION}`;
+    const rtvPath = path.join(outputDir, 'org-cdn/rtv', rtvNumber);
+    const channelPartialConfig = {
+      v: rtvNumber,
+      type: channelConfig.type,
+      ...require(`../../global-configs/${channelConfig.configBase}-config.json`),
+    };
+
+    allPrependPromises.push(
+      ...TARGETS_TO_CONFIG.map(async (target) => {
+        const targetPath = path.join(rtvPath, target.file);
+        const channelConfig = JSON.stringify({
+          ...channelPartialConfig,
+          ...target.config,
+        });
+
+        const contents = await fs.readFile(targetPath, 'utf-8');
+        return fs.writeFile(
+          targetPath,
+          `self.AMP_CONFIG=${channelConfig};/*AMP_CONFIG*/${contents}`
```

Comment:
Add this file to this allowlist to fix the presubmit error:

https://github.com/ampproject/amphtml/blob/e26bb7d2c8c4dd05d4b9bd44733a3051da3b8b92/build-system/tasks/presubmit-checks.js#L642-L667

Reply:
Done